modules/local/fusionreport/download/main.nf

```diff
@@ -4,7 +4,7 @@ process FUSIONREPORT_DOWNLOAD {
 
     // Note: 2.7X indices incompatible with AWS iGenomes.
     conda "bioconda::star=2.7.9a"
-    container "docker.io/clinicalgenomics/fusion-report:2.1.5p2.1"
+    container "docker.io/clinicalgenomics/fusion-report:2.1.5p3"
```

Comment:
what did you change in the container @rannick ? Also, did you test this works?

Reply:
Yes I tested the download of fusionreport references

Reply:
I modified this line: https://github.com/Clinical-Genomics/fusion-report/blob/e21f5f72eed0f1b80fd16d4741809fa0f3c7470a/fusion_report/common/net.py#L207
so that the files in the folder _MACOSX are not considered